packages/rspack/src/profiler.ts

```diff
@@ -0,0 +1,43 @@
+import { performance } from 'perf_hooks';
+import { spawn, ChildProcess } from 'child_process';
+import { Session } from 'inspector';
+import fs from 'fs';
+
+const profileIndex = process.argv.indexOf('--profile');
+
+function start(): ChildProcess {
+    const rspackProcess = spawn('node', ['--cpu-prof', './path-to-rspack/rspack', '-c', 'rspack.config.js'], {
```

Comment:
I would need to get the rspack binary path, do I need to grab this from the config file

Reply:
cc @hardfist

Reply:
you don't need to spawn a process here, profile session can be trigger by node api like https://github.com/vitejs/vite/blob/92b7fbb1c9526d68a05e370cc246c75d9e1a6e5e/packages/vite/bin/vite.js#L53